apps/api/src/donations/dto/donation-metadata.dto.ts

```diff
@@ -12,7 +16,7 @@ export class DonationMetadataDto {
   @ApiProperty()
   @Expose()
   @IsObject()
+  @Type(() => TExtraData)
   @IsOptional()
-  //eslint-disable-next-line
-  extraData: any
+  extraData: TExtraData | undefined
```

Comment:
might be good to add an example here of the expected structure, because my old mind fails to compile it ;)